entities/weapons/pocket/cl_menu.lua

```diff
@@ -61,7 +61,7 @@ function DarkRP.openPocketMenu()
     reload()
     frame:SetSkin(GAMEMODE.Config.DarkRPSkin)
 end
-
+usermessage.Hook("PocketMenu", DarkRP.openPocketMenu)
```

Comment:
There are existing usermessages that I cannot convert to net messages, but usermessages _are_ deprecated. Could you change this to use net messages rather than usermessages? Thanks!